src/lib/mina_base/test/account/account.ml

```diff
@@ -0,0 +1,273 @@
+open Core_kernel
+open Currency
+open Mina_base
+open Mina_numbers
+open Unsigned
+
+let gen_timing = Account.gen_timing Balance.max_int
+
+let gen_timing_with_necessary_vesting =
+  let open Quickcheck in
+  let open Account.Timing.As_record in
+  Account.gen_timing Balance.max_int
+  |> Generator.filter ~f:(fun t ->
+         let open Balance in
+         t.initial_minimum_balance - t.cliff_amount
+         |> Option.value_map ~default:false ~f:(fun to_vest -> to_vest > zero) )
```

Comment:
I'm pretty sure just doing `initial_minimum_balance > cliff_amount` is the same as this.